XenAdmin/Alerts/Types/ClientUpdateAlert.cs

```diff
@@ -69,16 +75,11 @@ public override Action FixLinkAction
 
         public override string HelpID => "XenCenterUpdateAlert";
 
-        public bool Downloadable { 
-            get {
-                // TODO: Implement proper check in CP-31587
-                return true;
-            }
-        }
-
-        public string Checksum = string.Empty;
+        // TODO: Implement proper check in CP-31587
+        //TODO: Check if path is url or msi to set downloadable

```

Comment:
I haven't changed this because I'm not very sure what check it refers to, could you please clarify?

Reply:
Ok, I'll have one more look at it. If it's not needed, are you happy for me to remove it?

Reply:
Absolutely

Reply:
Ok, I removed it.